tests/test_cpp.rs

```diff
@@ -241,3 +240,10 @@ pub fn test_from_cpp_ms(cl: &Client, testdata: &TestData) {
         assert!(num_conf > 0);
     }
 }
+
+#[test]
+fn tests_from_cpp() {
+    let cl = &setup::setup().client;
```

Comment:
I am a little puzzled in understanding how this test could work, since once `BitcoinD` struct goes out of scope, the node is stopped

Reply:
setup returns BitcoinD. It never goes out of scope.